image_option_api_test.go

```diff
@@ -8,7 +8,7 @@ import (
 )
 
 func Test_WithBuiltinImageEncoder(t *testing.T) {
-	oo := *_defaultOutputOption
```

Comment:
there is no need to keep copying semantics, just
```go
oo := defaultOutputOption()
```
this change also need to apply to the following codes